ROADMAP.md

```diff
@@ -69,25 +60,18 @@ These are goals achievable within 6-12 months, tentatively done by mid 2023.
   When most of the API can be used asynchronously.
 
 
-- **Beta availability of the extension in k6 Cloud for all users.**<br>
-
-  *How will we achieve this?*<br>
-  The deployment should be optimized and the extension thoroughly tested before making it available to all users. Frontend changes should be done at this point, and usage costs (CPU, RAM, storage) and pricing details should be determined, followed by public announcements of the availability. Features such as screen capture, video recording, downloading, and file uploading should be available.
-
-  *Definition of Done*<br>
-  When all users are eligible to upgrade to a plan that includes browser testing.
+Mid-term goals
+--------------
 
+These are goals tentatively achievable by Q1 2024.
 
-- **Merge the extension into the main k6 repository.**<br>
-  k6 and Grafana consider browser testing to be another strategy under the testing umbrella, along with performance, contract, and other types of testing that modern web applications benefit from. As such, the scope of the k6 tool will expand to include functional/E2E testing using browser automation, with the ultimate goal of merging the xk6-browser extension into the main k6 repository as a core JS module.
+- **Global availability of the extension in k6 Cloud for all users.**<br>
```

Comment:
Does this contradict with [this item](https://github.com/grafana/xk6-browser/pull/658/files#diff-683343bdf93f55ed3cada86151abb8051282e1936e58d4e0a04beca95dff6e51R35)? Like I discussed [in this other comment](https://github.com/grafana/xk6-browser/pull/658/files#r1031460086), won't we make k6 browser available for all users already?

Reply:
It will be available, possibly with limitations in terms of resource usage. The GA would mean that extension is not experimental anymore and the API is considered to be stable (see the definition of done)